msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -923,20 +923,10 @@ private static void create(@NonNull final PublicClientApplicationConfiguration c
 
         final CommandParameters params = CommandParametersAdapter.createCommandParameters(config, config.getOAuth2TokenCache());
 
-        final BaseController controller;
-        try {
-            controller = MSALControllerFactory.getDefaultController(
-                    config.getAppContext(),
-                    config.getDefaultAuthority(),
-                    config);
-        } catch (MsalClientException e) {
-            listener.onError(e);
-            return;
-        }
-
         final GetDeviceModeCommand command = new GetDeviceModeCommand(
                 params,
-                controller,
+                new MSALControllerFactory(config).
```

Comment:
nit: controllerfactory instantiation calls similar this one in this class are stylistically inconsistent; I personally like 1368-1369 the best (1213-1214 is similar, but feels like the parentheses are off?)